packages/lib-classifier/src/plugins/tasks/SurveyTask/components/SurveyTask.js

```diff
@@ -1,4 +1,4 @@
-import { Box, Drop } from 'grommet'
+import { Stack } from 'grommet'
```

Comment:
Stack covers the Done or equivalent buttons if the content is long enough, feels close (the same?) to the original design, though clicking outside of and other aspects Drop provided aren't included in Stack, thought would include for initial thoughts, though can easily remove and just use Box, let me know what you think.